kvbc/src/reconfiguration_kvbc_handler.cpp

```diff
@@ -234,8 +234,9 @@ bool StateSnapshotReconfigurationHandler::handle(const concord::messages::StateS
              "StateSnapshotRequest(participant ID = " << cmd.participant_id << "): using existing last checkpoint ID: "
                                                       << last_checkpoint_desc->checkPointId_);
   } else {
-    const auto checkpoint_id =
-        DbCheckpointManager::instance().createDbCheckpointAsync(sequence_number, timestamp, std::nullopt);
+    // Here we don't save the shared data about the db checkpoint creation. This is because this snapshot is not meant
```

Comment:
So what should it be used for?

Reply:
It is used for retrieving the blockchain state for a client node. A client node doesn't care about the bft data, only about the blockchain itself